README.md

```diff
@@ -88,12 +88,12 @@ The only required section is `site_info`
 ```
 This section is used to setup your WordPress site for the first time. You can change most of these settings with the WordPress admin once you have logged in after setup.
 
-You can define a specific version of WordPress that you would like to use with `wordpress_version`. If you do not specify, it will use the latest release of WordPress.
+You can define a specific version of WordPress that you would like to use by adding `"wordpress_version": VERSION_NUMBER` on line above the "site_info" section. If you don't add this line, cloud.gov will use the latest release of WordPress.
```

Comment:
I'm not crazy about this, because it means this example may stop working as the upstream version of WordPress drifts. It's also good practice to know _exactly_ what you're deploying and control when you accept and deploy changes from upstream. I'm not opposed to it; just feedback.